package.json

```diff
@@ -56,14 +56,18 @@
     "lint": "eslint src test",
     "lint:fix": "eslint --fix src test",
     "clean:node": "rm -rf index.js lib",
+    "clean:node.win": "RMDIR /S /Q lib && ERASE /F /Q index.js",
     "clean:browser": "rm -rf validator*.js",
+    "clean:browser.win": "ERASE /F /Q validator*.js",
     "clean": "npm run clean:node && npm run clean:browser",
+    "clean.win": "npm run clean:node.win && npm run clean:browser.win",
```

Comment:
PR LGTM but i can see that you added some windows specific script commands to replace ```rm -rf``` may be we can avoid adding them by using [rimraf](https://www.npmjs.com/package/rimraf)
@profnandaa what's for you the best approach to add windows compatibility in the development process between adding specific commands or using an external dependency ?